pallets/lockdown-mode/src/mock.rs

```diff
@@ -119,8 +119,12 @@ impl pallet_lockdown_mode::Config for Test {
 	type WeightInfo = pallet_lockdown_mode::weights::SubstrateWeight<Test>;
 }
 
-// Build genesis storage according to the mock runtime.
-pub fn new_test_ext() -> sp_io::TestExternalities {
-	let storage = system::GenesisConfig::default().build_storage::<Test>().unwrap();
+pub fn new_test_ext(activated: bool) -> sp_io::TestExternalities {
```

Comment:
Same remark as previous one. I find `initial_status` would be clearer.

Reply:
Renamed.